### DISCUSSION TITLE
Metrics cannot be generated... error

### DISCUSSION THREAD

**Discussion by Crispy-Cream** [lowlighter/metrics#503] [Q&A] [answered]
"Metrics cannot be generated because the following plugins are not available on this web instance: music"

What do I do to get the music plugin and others working

**Comment by spenserblack** [comment, marked as answer]
You might want to set up a scheduled task instead, which can use all plugins. On the [metrics site](https://metrics.lecoq.io/), once you fill in your username and select the plugins you want, you can go to the "Action code" tab. You can copy and paste the content into `.github/workflows/<workflow name>.yml`.
